README.md

```diff
@@ -1,13 +1,13 @@
 # Design Handoff
 
-Replace this readme with your own information about your project.
+This React App is designed to showcase how to implement and integrate various design elements such as color, typography, and hierarchy in web design using Styled Components as a CSS framework.
```

Comment:
I like that you make clear and concise read-me's - they help a lot :) 
I think you made a great job with the hero-section and the section below. Given the size of this project and the limited time/size of team to work with combinated with Easter week and that Technigo promised to scale the project because of this (but many experienced the opposite after the discussions I've read) - I think you made a great job!